src/name_service/delivery/utils.clj

```diff
@@ -0,0 +1,47 @@
+(ns name-service.delivery.utils
+  (:require [cheshire.core :as json]
+            [io.pedestal.http :as bootstrap]
+            [io.pedestal.http.body-params :as body-params]
+            [name-service.core.action :as action]
+            [name-service.delivery.http :as http]))
+
+(defn map-key-first [map]
```

Comment:
This name is very unreadable and misleading. This function takes a map of one key and returns
this key as a string. The name suggests that the function is mapping some "first" key. Mapping to what?

Better name would be something like `get-first-key-as-str` or `first-key-to-str`. It's longer, but clearly states intent.
Same applies to `map-value-first`.